### PR TITLE
Set selection to use Matlab characteristics, add simple guide star selection, use guide star SNR in output

### PR DESCRIPTION
**Pull request by jeanconn**
These changes set the mini_sausage to use at least *more* of the selection parameters for the stages directly from the Matlab characteristics JSON file.  Guide star selection has also been added, but instead of optimizing the selection of stars in the "last" stage by using a merit calculation based on the ACA_ERROR_ARRAY and distance (SAUSAGE method), this code just selects the brightest "OK" stars from the last attempted stage to fill the requested number of candidate entries (which is 9 for guide from the characteristics, though only the first n will be returned by the higher-level selection routine).

The changes in check_sausage.py are not important.  I just used it to explore some of the worst matches (comparing catalogs as retrieved from starcheck and catalogs as they would be created by mini_sausage), but did not use it to generate a quantitative report/analysis.  Qualitatively, the guide star selection matches expectations.  My conclusion is that in the case of many candidates in the last stage that the two selection methods can vary a lot, but we probably don't care as much in those situations (too many good guide stars is a great problem to have).  I found that other causes of mismatches were due to the stars being excluded by SAUSAGE due to the direct catalog search (with very faint spoilers doing the spoiling).  While the mini_sausage has code to replicate that direct catalog search, I have mostly been using the miniagasc, which doesn't have the faint stars *in* it to cause the same spoiling.  Use of a complete AGASC 1.6 in mini_sausage resulted in qualitatively the same rejections as the stars rejected in SAUSAGE.

My plan will likely be to continue to use the miniagasc for work with the evaluator, though it might be better to just remove the "direct catalog search" in the mini_sausage selection instead of crippling it via a catalog without faint stars.

### CODE REVIEW

File: aca_lts_eval.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def t_ccd_for_attitude(ra, dec, cycle, detector, too, y_offset=0, z_offset=0,
         raise ValueError("No stars found in 3 degree radius of {} {}".format(ra, dec))
 
     # get mag errs once for the field
-    cone_stars['mag_one_sig_err'], cone_stars['mag_one_sig_err2'] = mini_sausage.get_mag_errs(cone_stars)
+    #cone_stars['mag_one_sig_err'], cone_stars['mag_one_sig_err2'] = mini_sausage.get_mag_errs(cone_stars)
```
Comment (jeanconn) [inline aca_lts_eval.py:382]: I think this can safely be uncommented (the mag errors are not specific to acquisition or guide selection).

**Comment by jeanconn** [comment]
Also, this makes no improvements to the overall spaghetti code.

**Comment by jeanconn** [comment]
What kind of validation does this need before promotion to operational use?

**Comment by jeanconn** [comment]
@taldcroft What do we need for approval for promotion of the updated ACA lts eval software with guide star selection (and calculation of 4th brightest star derived temperature limit).  Do we just need SOT MP to bring their tool change (to read a different column) for approval, or is even that outside LR scope?

**Comment by taldcroft** [comment]
I don't think any of this is controlled by LR board, but you can check with SOT MP.  (Unless you know the name of their script that is changing, then you can look on the FDB SOT MP checklist which lists controlled tools).

**Pull request merged**